modules/storage/cosmos-db/main.bicep

```diff
@@ -1,99 +1,353 @@
-@description('Deployment Location')
-param location string
+@description('Deployment region name. Default is the location of the resource group.')
+param location string = resourceGroup().location
 
-@description('Prefix of Cosmos DB Resource Name')
-param prefix string = enableCassandra ? 'coscas' : 'cosmos'
+@description('The bakend API type of Cosmos DB database account. The API selection cannot be changed after account creation. Possible values: "cassandra", "gremlin", "mongodb", "sql", "table".')
+@allowed([ 'cassandra', 'gremlin', 'mongodb', 'sql', 'table' ])
+param backendApi string = 'cassandra'
```

Comment:
```suggestion
param backendApi string = 'sql'
```

Reply:
lets set default values to regular comsos

Reply:
Changed default backend type to `sql`.